00-programming-fundamentals/intro-to-ruby-mine.md

```diff
@@ -0,0 +1,43 @@
+# Introduction to RubyMine
+
+RubyMine will be the application we use to read and write Ruby code from now on. It is referred to as a Ruby Integrated Development Environment (or Ruby IDE, for short). In other words, it's more than a text editor.
+We have started to get familiar with VS Code, which we will use again later on in the curriculum, but before we spend much time with Ruby we want to get comfortable with a heftier tool that understands how to read Ruby and is able to help us _debug_ Ruby programs (something we will learn how to do very soon). 
+
+VS Code is to RubyMine what a paper map is to a GPS. We _can_ build big projects in Ruby with VS Code. However, RubyMine is a tool that enables us to focus less on missing that right turn so that we can focus more on understanding the big picture of where we want to go.
+
+### Adoption at Ada
+
+Full disclosure: previous cohorts have used VS Code for the full Ruby curriculum. C14 is the first cohort to use RubyMine. We are making the switch because RubyMine will give us the ability to debug in a way we aren't able to with VS Code, which is an important skill we will learn and practice throughout the curriculum.
+
+Please note that if there are any mentions of `VS Code` or the command `code .` during the Ruby portions of the curriculum, that the document is outdated. If something in the Ruby curriculum is talking about VS Code, it is probably meant to be talking about RubyMine.
+
+## Awesome Keyboard Shortcuts
+To view all the keyboard shortcuts, navigate to `Code > Preferences > Keymap`
+Below are a few you'll likely use.
+
+Shortcut | Effect
+---      | ---
+`Double ⇧`  | Search Everywhere. Find anything related to RubyMine or your project and open it, execute it, or jump to it.
+`⇧⌘A`     | Find a command and execute it. Perfect when you don't remember the shortcut for something else
+`⌘B` or `⌘click`     | Go to declaration. Navigate to the initial declaration of the instantiated class, called method, or field. This is one of the powerful things IDEs can do for us!
+`⌘C` (nothing highlighted)    | Copy the current line
+`⌘/`     | Comment out the current line
+`^⌥I`   | Auto Fix Your Indentation
+`⌘ ←(Backspace)`    | Delete the current line
+
+Fun fact: RubyMine makes it really easy for us to change these shortcuts and make our own new shortcuts! If any of these shortcuts feel hard to remember, maybe try switching them to something that makes more sense to you personally.
+
+Remember that normal macOS keyboard shortcuts will work here too!
+
+Shortcut | Effect
+---      | ---
+`⌥left` & `⌥right` | Jump left or right by one word
+`⌘left` & `⌘right` | Jump to the beginning or the end of the current line
+`⌘up` & `⌘down` | Jump to the top or the bottom of the document
+`⌘⇧[` & `⌘⇧]` | Switch tabs left or right
+`⌘,`     | Open the settings page
+`⌘w`     | Close the current tab
+`⌘⇧w`    | Close the current window (all tabs)
```

Comment:
More kbd tags, also I split all of your paired commands into their own rows in the for clarity table just one after the other.

```suggestion
<kbd>⌥</kbd> + <kbd>←</kbd> | Jump left by one word
<kbd>⌥</kbd> + <kbd>→</kbd> | Jump right by one word
<kbd>⌘</kbd> + <kbd>←</kbd> | Jump to the beginning of the current line
<kbd>⌘</kbd> + <kbd>→</kbd> | Jump to the end of the current line
<kbd>⌘</kbd> + <kbd>↑</kbd> | Jump to the top of the document
<kbd>⌘</kbd> + <kbd>↓</kbd> | Jump to the bottom of the document
<kbd>⌘</kbd> + <kbd>shift</kbd> + <kbd>[</kbd> | Switch tabs left
<kbd>⌘</kbd> + <kbd>shift</kbd> + <kbd>]</kbd> | Switch tabs right
<kbd>⌘</kbd> + <kbd>,</kbd>  | Open the settings page
<kbd>⌘</kbd> + <kbd>w</kbd>     | Close the current tab
<kbd>⌘</kbd> + <kbd>shift</kbd> + <kbd>w</kbd>    | Close the current window (all tabs)
```